### PR TITLE
Further bugfixes for 6.3 RC2

### PR DESCRIPTION
**Pull request by tellthemachines**
## What?

Adds the following bug fixes to the 6.3 release branch:

[Sidebar: Restore Back buton 'go to parent' functionality](https://github.com/WordPress/gutenberg/pull/52910) 
[Disambiguate "Import" button string.](https://github.com/WordPress/gutenberg/pull/52907)
[Site Editor: Fix the template parts link on the list page](https://github.com/WordPress/gutenberg/pull/52891) 
[Check if object exists before accessing its properties.](https://github.com/WordPress/gutenberg/pull/52870) 
[Distraction Free: Add missing command in site editor](https://github.com/WordPress/gutenberg/pull/52868) 
[Distraction Free Keyboard Shortcut: Fix notices in site editor](https://github.com/WordPress/gutenberg/pull/52867)
[Global styles revisions: display text if no revisions are found](https://github.com/WordPress/gutenberg/pull/52865)
[Image: Use the correct method for caption class in recent deprecation](https://github.com/WordPress/gutenberg/pull/52853)
[Fix image block v6 deprecation](https://github.com/WordPress/gutenberg/pull/52822) 
[Return focus more from focus return hook](https://github.com/WordPress/gutenberg/pull/52710) 
[My patterns page: Increase color contrast for the toggle group](https://github.com/WordPress/gutenberg/pull/52678)
[Distraction Free: Fix conflict with showListViewByDefault preference](https://github.com/WordPress/gutenberg/pull/52914) 
[Site Editor: Don't navigate to the patterns in Template Parts mode](https://github.com/WordPress/gutenberg/pull/52884)
[Site Editor: Open template parts from the list page in canvas view mode](https://github.com/WordPress/gutenberg/pull/52916) 
[ResizableFrame: Account for window resizing](https://github.com/WordPress/gutenberg/pull/52697)

### CODE REVIEW

**Review by ramonjd — APPROVED** [review]
Tested all commits in the list according to their PR descriptions

LGTM

🙇🏻

**Comment by Mamaduka** [comment]
@tellthemachines, @ramonjd, is there a time to include one or two more PRs?

https://github.com/WordPress/gutenberg/pull/52916 and https://github.com/WordPress/gutenberg/pull/52884?

**Comment by ramonjd** [comment]
> is there a time to include one or two more PRs?

Both approved. Just waiting for CI to pass on https://github.com/WordPress/gutenberg/pull/52916

**Comment by Mamaduka** [comment]
Thanks and sorry for last minute PRs 🙇

**Comment by ramonjd** [comment]
Latest commits LGTM

File: packages/block-library/src/footnotes/index.php [hunk 1]
```diff
@@ -83,136 +83,132 @@ function register_block_core_footnotes() {
...
+ * @param int $revision_id The revision ID.
+ */
+function wp_save_footnotes_meta( $revision_id ) {
+	$post_id = wp_is_post_revision( $revision_id );
+
+	if ( $post_id ) {
+		$footnotes = get_post_meta( $post_id, 'footnotes', true );
+
+		if ( $footnotes ) {
+			// Can't use update_post_meta() because it doesn't allow revisions.
```
Comment (spacedmonkey) [inline packages/block-library/src/footnotes/index.php:100]: ???

Comment (Mamaduka) [reply packages/block-library/src/footnotes/index.php:100]: See https://github.com/WordPress/wordpress-develop/blob/6.2/src/wp-includes/post.php#L2518-L2519

**Pull request merged**